hvplot/plotting/scatter_matrix.py

```diff
@@ -38,19 +47,99 @@ def scatter_matrix(data, c=None, chart='scatter', diagonal='hist', alpha=0.5, **
     diagonal = HoloViewsConverter._kind_mapping[diagonal]
     chart = HoloViewsConverter._kind_mapping[chart]
 
+    if rasterize:
+        if dynspread or spread:
+            if hd.ds_version < '0.12.0':
+                raise RuntimeError('Any version of datashader ' +
+                                    'less than 0.12.0 does not support ' + 
+                                    'rasterize with dynspread or spread')
+    #remove datashade kwds
+    if datashade or rasterize:
+        ds_kwds = {}
+        if 'aggregator' in kwds:
+            ds_kwds['aggregator'] = kwds.pop('aggregator')
+        
+    #remove dynspread kwds
```

Comment:
```suggestion
    #remove dynspread kwds
    sp_kwds={}
```

(Needed for changes below)